index.php

```diff
@@ -114,6 +114,15 @@
 
 // Set HTTP-Headers
 header('Content-Type: text/html; charset=utf-8');
+header('X-Frame-Options: sameorigin');
+header('X-XSS-Protection: 1; mode=block');
+header('X-Content-Type-Options: nosniff');
+header('Referrer-Policy: no-referrer');
```

Comment:
What is the purpose of sending no referrer at all?
Often, the referrer is pretty useful, especially when this website is sending bad requests to some other server.

I am more in favor of using the default policy (`[strict-origin-when-cross-origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy#strict-origin-when-cross-origin)`) here.

Reply:
```suggestion
header('Referrer-Policy: strict-origin');
```